src/main/java/org/sagebionetworks/bridge/exporter/util/BridgeExporterUtil.java

```diff
@@ -35,6 +39,10 @@
     public static final String CONFIG_KEY_TIME_ZONE_NAME = "time.zone.name";
     public static final String CONFIG_KEY_RECORD_ID_OVERRIDE_BUCKET = "record.id.override.bucket";
     public static final String CONFIG_KEY_SQS_QUEUE_URL = "exporter.request.sqs.queue.url";
+    
+    private static final ObjectMapper MAPPER = new ObjectMapper();
```

Comment:
Is this still needed?

If it's stil needed, we generally use DefaultObjectMapper.INSTANCE instance of creating a new ObjectMapper in every class.

Reply:
Removed